### ISSUE TITLE
Digit Differ - Prediction after 2 or 3 rounds

### ISSUE THREAD

**Issue by sumanthvishnu** [binary-com/binary-bot#938]
Hi,

I am trying to create an xml for the below mentioned logic

If the last digit is 0, then check for the below logic,

Whether next 5 ticks with 0 as the last digit in the 5th tick
1234.0078
1362.7723
1462.6734
1323.8321
1882.2310 <--- if yes

Then, next 5 ticks with 0 as the last digit in the 5th tick
1876.0078
1227.7723
1823.6734
1827.8321
1992.2310 <--- if yes, then purchase with '0' as last digit prediction in the next 5 ticks

If the result is a win, then start the sequence again from the beginning.

If the result is a loss, then apply martingale amount(stake*110%) and purchase the immediate deal with '0' as prediction in next 5 ticks.

After this, if the result is a win, then start the sequence again from the beginning.
And if it is a loss, stop everything.

I am finding it difficult to write this logic as I am very new to XML. Can someone pls help.

In short i want the last digit prediction to be made a couple of times continuously before applying a martingale kind of strategy for digit differs.

Thanks in advance :)

**Comment by sinung21** [comment]
Maybe this can help u bro
[LD 0 5TICK MORE.zip](https://github.com/binary-com/binary-bot/files/2545124/LD.0.5TICK.MORE.zip)

**Comment by nhatthaihoaithu** [comment]
THANKS VERY MUCH.IT IS WORKING VERY GOOD.
